### PR TITLE
Use plone.app.multilingual conditionally

### PR DESCRIPTION
**Pull request by folix-01**

### CODE REVIEW

File: src/plone/restapi/services/configure.zcml [hunk 1]
```diff
@@ -46,10 +46,12 @@
...
       />
-  <include
-      package=".multilingual"
-      zcml:condition="have plone-5"
-      />
+  <configure zcml:condition="installed plone.app.multilingual">
+    <include
+        package=".multilingual"
+        zcml:condition="have plone-5"
+        />
```
Comment (jensens) [inline src/plone/restapi/services/configure.zcml:53]: While at it, we can drop the outdated condition too.
```suggestion
    <include
        package=".multilingual"
        />
```

**Review by jensens — CHANGES_REQUESTED** [review]

**Comment by jensens** [comment]
Why does it touch many unrelated files?

**Comment by erral** [comment]
Why is this needed?

plone.restapi master is already Plone 5.x only which already includes p.a.multilingual as a dependency.

**Comment by folix-01** [comment]
> Why does it touch many unrelated files?

Because of black I think

**Comment by folix-01** [comment]
> Why is this needed?
> 
> plone.restapi master is already Plone 5.x only which already includes p.a.multilingual as a dependency.

Because we need to remove il from the Products.CMFPlone

**Comment by jensens** [comment]
We remove it as a dep of CMFPlone and add it as a dep on Plone. Thus it becomes a core-addon. Architecture wise this is favorable. At the end, for the user, nothing changed.

**Review by jensens — APPROVED** [review]

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**